### DISCUSSION TITLE
How to receive the delivery status?

### DISCUSSION THREAD

**Discussion by progressify** [ibnux/Android-SMS-Gateway#37] [Q&A]
There is a way to receive (also in asynchronous) the delivery status of the message?
(sent, delivered, etc)

**Comment by ibnux** [comment]
yes it will send callback with type=delivered using post

https://github.com/ibnux/Android-SMS-Gateway/blob/729190f5a641e19c165516847d1ddc3bd74e5cf6/backend/index.php#L3-L14

**Reply by progressify** [reply]
I am sorry, I am not understanding how it works, can you be clearest? 🤔
